concepticondata/conceptlists/Morse-1996-375.tsv

```diff
@@ -0,0 +1,376 @@
+Morse-1996-375-32	32	hueso	bone	*	1394	BONE
+Morse-1996-375-33	33	sangre	blood		946	BLOOD
+Morse-1996-375-34	34	corazón	heart	*	1223	HEART
+Morse-1996-375-35	35	pulmones	lungs		688	LUNG
+Morse-1996-375-36	36	pene	penis		1222	PENIS
+Morse-1996-375-37	37	vulva	vulva		1837	VULVA
+Morse-1996-375-38	38	hombre	man	*	1554	MAN
+Morse-1996-375-39	39	varón	man		2106	MALE PERSON
+Morse-1996-375-40	40	mujer	woman	*	962	WOMAN
+Morse-1996-375-41	41	gente	people	*	789	PEOPLE
+Morse-1996-375-42	42	marido	husband	*	1200	HUSBAND
+Morse-1996-375-43	43	esposa	wife	*	1199	WIFE
+Morse-1996-375-44	44	papá	father	*	1217	FATHER
+Morse-1996-375-45	45	mamá	mother	*	1216	MOTHER
+Morse-1996-375-46	46	criatura	baby infant		1289	BABY
+Morse-1996-375-47	47	viejo	old man		406	OLD MAN
+Morse-1996-375-48	48	agua	water	*	948	WATER
+Morse-1996-375-49	49	rio	river	*	666	RIVER
+Morse-1996-375-50	50	isla	island		622	ISLAND
+Morse-1996-375-51	51	lago	lake	*	624	LAKE
+Morse-1996-375-52	52	pantano	swamp		1145	SWAMP
+Morse-1996-375-53	53	fuente, ojo de agua	spring of water		849	SPRING (OF WATER)
+Morse-1996-375-54	54	caida de agua	waterfall		943	WATERFALL
+Morse-1996-375-55	55	catarata, raudal	rapids		1670	RAPIDS
+Morse-1996-375-56	56	candela, fuego	fire	*	221	FIRE
+Morse-1996-375-57	57	ceniza	ash	*	646	ASH
+Morse-1996-375-58	58	carbón	charcoal		1188	CHARCOAL
+Morse-1996-375-59	59	humo	smoke	*	778	SMOKE (EXHAUST)
+Morse-1996-375-60	60	leña	firewood		10	FIREWOOD
+Morse-1996-375-61	61	cielo	sky	*	1732	SKY
+Morse-1996-375-62	62	lluvia	rain	*	658	RAIN (PRECIPITATION)
+Morse-1996-375-63	63	viento	wind	*	960	WIND
+Morse-1996-375-64	64	sol	sun	*	1343	SUN
+Morse-1996-375-65	65	luna	moon	*	1313	MOON
+Morse-1996-375-66	66	estrella	star	*	1430	STAR
+Morse-1996-375-67	67	día	day	*	1225	DAY (NOT NIGHT)
+Morse-1996-375-68	68	noche	night	*	1233	NIGHT
+Morse-1996-375-69	69	trueno	thunder		1150	THUNDER
+Morse-1996-375-70	70	relámpago	lightning		1654	LIGHTNING
+Morse-1996-375-71	71	arco iris	rainbow		1733	RAINBOW
+Morse-1996-375-72	72	tierra	earth	*	1228	EARTH (SOIL)
+Morse-1996-375-73	73	piedra	stone	*	857	STONE
+Morse-1996-375-74	74	arena	sand	*	671	SAND
+Morse-1996-375-75	75	casa	house		1252	HOUSE
+Morse-1996-375-76	76	techo	roof		769	ROOF
+Morse-1996-375-77	77	puerta	doorway		1567	DOOR
+Morse-1996-375-78	78	banco	seat, stool		2058	STOOL
+Morse-1996-375-79	79	estera	mat		195	MAT
+Morse-1996-375-80	80	hamaca	hammock		2019	HAMMOCK
+Morse-1996-375-81	81	catre, cama de barbacoa	bed		1663	BED
+Morse-1996-375-82	82	olla	cooking pot		3653	COOKING POT
+Morse-1996-375-83	83	roza, chagra	cultivated clearing			
+Morse-1996-375-84	84	caserío	village		930	VILLAGE
+Morse-1996-375-85	85	camino, sendero, trocha	path, trail	*	2252	PATH
+Morse-1996-375-86	86	red	fish net		388	FISHNET
+Morse-1996-375-87	87	anzuelo	fish hook		1074	FISHHOOK
+Morse-1996-375-88	88	hacha	axe		677	AXE
+Morse-1996-375-89	89	cuchillo	knife		1352	KNIFE
+Morse-1996-375-90	90	canoa	canoe		1970	CANOE
+Morse-1996-375-91	91	remo	paddle		234	PADDLE
```

Comment:
Isn't a remo an OAR? At least the Wikipedia article implies so if I understand correctly.

Reply:
You are right, I will change it.